src/index.js

```diff
@@ -80,10 +80,17 @@ exports.sourceNodes = async (
   const fetchActivity = reporter.activityTimer(`Fetched Strapi Data`);
   fetchActivity.start();
 
-  const contentTypes = (options.contentTypes || []).map(contentTypeToTypeInfo);
+  const collectionTypes = (options.collectionTypes || []).map(contentTypeToTypeInfo);
   const singleTypes = (options.singleTypes || []).map(singleTypeToTypeInfo);
 
-  const types = [...contentTypes, ...singleTypes];
+  // Let users know that options.contentTypes is deprecated
+  if (options.contentTypes) {
```

Comment:
Let's not keep some deprecation warning in the code since we are releasing a new major version. We will add this to the release changelog instead